src/Take.Blip.Builder/Actions/ProcessCommand/ProcessCommandAction.cs

```diff
@@ -49,8 +52,10 @@ public async Task ExecuteAsync(IContext context, JObject settings, CancellationT
 
         private Command ConvertToCommand(JObject settings)
         {
+            var regex = new Regex(SERIALIZABLE_PATTERN, RegexOptions.Compiled | RegexOptions.IgnoreCase, _configuration.RegexTimeout);
+
             if (settings.TryGetValue(Command.TYPE_KEY, out var type)
-                && Regex.IsMatch(type.ToString(), SERIALIZABLE_PATTERN)
+                && regex.IsMatch(type.ToString())
```

Comment:
You don't need to initialize the regex, you can still use the static method:

https://docs.microsoft.com/pt-br/dotnet/api/system.text.regularexpressions.regex.ismatch?view=net-5.0

Reply:
Done

Reply:
Done